core/src/main/java/io/aiven/klaw/controller/UtilController.java

```diff
@@ -69,4 +76,31 @@ public ResponseEntity<Map<String, String>> resetMemoryCache(
   public void shutdownApp() {
     utilControllerService.shutdownContext();
   }
+
+  /**
+   * @param requestEntityType topic, acl, schema, connector, user
+   * @param requestStatus requests in different status created/deleted/declined/approved/all
+   * @return RequestsCountOverview A count of each request entity type, and request status, and
+   *     overall count
+   */
+  /*
+     Get counts of all request entity types, and for their different status types
+  */
+  @Operation(
+      summary = "Get counts of all request entity types, and for their different status types",
+      responses = {
+        @ApiResponse(
+            content = @Content(schema = @Schema(implementation = RequestsCountOverview.class)))
+      })
+  @RequestMapping(
+      value = "/requests/requestEntityType/{requestEntityType}/requestStatus/{requestStatus}",
+      method = RequestMethod.GET,
```

Comment:
My only thought is that as this is a statistics api call we should include that in the path, and then use params to filter what we get back.
e.g.
/requests/overview?requestEntityType={requestEntityType}&?requestStatus={requestStatus}

/requests/statistics?requestEntityType={requestEntityType}&?requestStatus={requestStatus}

Reading the above I would read that I am getting the statistics/overview for requests filtered by the requestEntity type and RequestStatus.

Reply:
i agree, they should be params, instead of path variables